app/src/main/java/com/learningkotlin/pokedex/view/PokemonByTypeFragment.kt

```diff
@@ -0,0 +1,60 @@
+package com.learningkotlin.pokedex.view
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.learningkotlin.pokedex.R
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [PokemonByTypeFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class PokemonByTypeFragment : Fragment() {
+    // TODO: Rename and change types of parameters
+    private var param1: String? = null
+    private var param2: String? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            param1 = it.getString(ARG_PARAM1)
+            param2 = it.getString(ARG_PARAM2)
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_list_pokemon_type, container, false)
```

Comment:
Let implementation of onCreateView and onViewCreated, remove rest of boilerplate code including comments and not used functions. It will be good if viewBinding is implemented

Reply:
viewBinding was applied, fragments in general were cleaned up

Reply:
Ready to merge on master